src/commonMain/kotlin/com/notkamui/keval/AbstractSyntaxTree.kt

```diff
@@ -31,11 +31,11 @@ internal data class KevalBothOperator(
 /**
  * Represents a function
  *
- * @property arity is the arity of the function (how many arguments it takes)
+ * @property arity is the arity of the function (how many arguments it takes), flexible if null
```

Comment:
```suggestion
 * @property arity is the arity of the function (how many arguments it takes). If null, the function is variadic
```

Prefer using standard terms.